lib/events.js

```diff
@@ -825,3 +826,17 @@ function on(emitter, event, options) {
     iterator.return();
   }
 }
+
+const EventEmitterMixin = (Superclass) => {
+  class MixedEventEmitter extends Superclass {
+    constructor(...args) {
+      super(...args);
+      EventEmitter.call(this);
+    }
+  }
+  const protoProps = Object.getOwnPropertyDescriptors(EventEmitter.prototype);
+  delete protoProps.constructor;
+  Object.defineProperties(MixedEventEmitter.prototype, protoProps);
+  return MixedEventEmitter;
+};
+module.exports.EventEmitterMixin = EventEmitterMixin;
```

Comment:
This makes this a publicly accessible class … is that intentional? (I’d be very okay with it – but this would have to come with documentation, tests, etc. in that case)

Reply:
No, indeed, I didn't realize events.js had no internal counterpart, any suggestions where does this belong?